ibis/backends/datafusion/__init__.py

```diff
@@ -243,6 +243,50 @@ def read_parquet(
         self._context.register_parquet(table_name, path, **kwargs)
         return self.table(table_name)
 
+    def read_delta(
+        self, source_table: str | Path, table_name: str | None = None, **kwargs: Any
+    ) -> ir.Table:
+        """Register a Delta Lake table as a table in the current database.
+
+        Parameters
+        ----------
+        source_table
+            The data source. Must be a directory
+            containing a Delta Lake table.
+        table_name
+            An optional name to use for the created table. This defaults to
+            a sequentially generated name.
+        **kwargs
+            Additional keyword arguments passed to deltalake.DeltaTable.
+
+        Returns
+        -------
+        ir.Table
+            The just-registered table
+        """
+        source_table = normalize_filename(source_table)
+
+        table_name = table_name or gen_name("read_delta")
+
+        # Our other backends support overwriting views / tables when reregistering
+        # TODO: datafusion doesn't natively support Delta Lake tables at this point,
+        # so we need to add this later?
+        # self._context.deregister_table(table_name)
```

Comment:
or should this line be uncommented?

Reply:
I don't think whether support is native or not matters here, so this line should be uncommented.